images.go

```diff
@@ -15,42 +15,64 @@ type ImageStatus string
 
 // ImageStatus options start with ImageStatus and include all Image statuses
 const (
-	ImageStatusCreating      ImageStatus = "creating"
-	ImageStatusPendingUpload ImageStatus = "pending_upload"
-	ImageStatusAvailable     ImageStatus = "available"
+	ImageStatusAvailable          ImageStatus = "available"
+	ImageStatusCreating           ImageStatus = "creating"
+	ImageStatusPending            ImageStatus = "pending"
+	ImageStatusPendingReplication ImageStatus = "pending replication"
+	ImageStatusPendingDeletion    ImageStatus = "pending deletion"
+	ImageStatusPendingUpload      ImageStatus = "pending_upload"
+	ImageStatusReplicating        ImageStatus = "replicating"
```

Comment:
The statuses eventually included are slight different from the api spec: https://bits.linode.com/LinodeAPI/apinext/blob/7e1c7b51269678235c99925ee54e0fac571a1c88/apinext/maps/image.py#L60 We can remove `ImageStatusPendingUpload ` from here.

Reply:
Nice catch!

Reply:
Ohh actually, I didn't realize the ImageStatus and ImageRegionStatus should be two separate types. I'll get that fixed 👍